src/main/java/edu/nextstep/camp/racing/domain/Car.java

```diff
@@ -0,0 +1,32 @@
+package edu.nextstep.camp.racing.domain;
+
+public class Car {
+
+    private static final int MOVING_MIN_VALUE = 4;
+    private static final int MOVING_MAX_VALUE = 9 ;
+    private final String name;
+
+    private int position = 0;
+    public Car(String name) {
```

Comment:
```suggestion
    private int position = 0;
    
    public Car(String name) {
```
빈 공백 라인도 의미를 가지도록 주의해 사용하면 어떨까?